examples/oneapi/cpp/source/covariance/cov_dense_batch.cpp

```diff
@@ -25,8 +25,9 @@ int main(int argc, char const *argv[]) {
     const auto input_file_name = get_data_path("covcormoments_dense.csv");
 
     const auto input = dal::read<dal::table>(dal::csv::data_source{ input_file_name });
-    auto cov_desc = dal::covariance::descriptor{}.set_result_options(
-        dal::covariance::result_options::cov_matrix);
+    auto cov_desc = dal::covariance::descriptor{}
```

Comment:
its better to create a separate example cov_bias_dense_batch.cpp and the same thing for all modes: online and in samples also